routes/router.js

```diff
@@ -10,6 +11,5 @@ module.exports = function(app, contextRoot) {
     rootRouter.post('/addDemo', demoController.addDemo);
 
     //Please make any business router under the rootRouter, so that it will be easy for contextRoot config.
-    rootRouter.get('/message', messageController.message);
-    rootRouter.post('/sendMessage', messageController.sendMessage);
+    rootRouter.use('/message');
```

Comment:
You need to associate messageRouter to this path.
say, rootRouter.use('/message', messageRouter);